params/governance_params.go

```diff
@@ -112,61 +106,6 @@ var (
 	DefaultDeriveShaImpl             = uint64(0)     // Orig
 )
 
-func IsStakingUpdateInterval(blockNum uint64) bool {
```

Comment:
For tidying up instead of inlining, is it not better to refactor and reuse this function by adding an additional parameter for the interval value?